src/NLog.StructuredLogging.Json.Tests/EndToEnd/ViaLayout/WithFailingLayout.cs

```diff
@@ -57,7 +57,7 @@ public void ShouldSurviveWhenLayoutFails()
         }
 
         [Test]
-        [Ignore("todo: investigate why we don't get this failing prop out when in .Net core")]
+        [Category("NotInNetCore")]
```

Comment:
I don't know if it's caused by Linux or by .NET Core. We don't have a .net core test run in Appveyor.